docs/codeql/codeql-language-guides/customizing-library-models-for-java-and-kotlin.rst

```diff
@@ -291,3 +291,19 @@ The first four values identify the callable (in this case a method) to be modele
 - The fourth value ``()`` is the method input type signature.
 - The fifth value ``summary`` is the kind of the neutral.
 - The sixth value ``manual`` is the provenance of the neutral.
+
+.. _threat-models
+
+Threat models
```

Comment:
I also pushed a change directly to this section. I hope that's ok.

I think for now, it's best to explain how treat models interact with an analysis and how to use the existing threat models rather than go into detail about the implementation.